Plan: Fix missing reviewer prompt template that blocks orch review phase

# Plan: Fix missing reviewer prompt template that blocks orch review phase

**Status:** Draft
**Created:** 2026-04-13

## Requirements

- `orch-review.sh` must find its reviewer prompt template at `~/.degacore/scripts/ralph-item-reviewer-prompt.md`
- `/apply-core` must install `ralph-item-reviewer-prompt.md` alongside the other orchestrator scripts
- After fix, an orchestrator run must complete the full cycle: workers → review → SHIP/REVISE
- `shellcheck -e SC1091 scripts/orch-review.sh` passes (no regressions)

## Approach

The file `scripts/ralph-item-reviewer-prompt.md` already exists in the repo. The problem is that `commands/apply-core.md` never includes it in the orchestrator install list. The fix is a one-line addition to the apply-core manifest.

Two files to change:
1. **`commands/apply-core.md`** — Add `ralph-item-reviewer-prompt.md` to the orchestrator scripts install section (alongside `orch-run.sh`, `orch-engine.sh`, etc.)
2. **Immediate install** — Copy the file to `~/.degacore/scripts/` so the fix is live without re-running `/apply-core`

No changes to `orch-review.sh` — the path it references (`${SCRIPT_DIR}/ralph-item-reviewer-prompt.md`) is correct once the file is installed.

## Files to touch

| File | Change |
|------|--------|
| `commands/apply-core.md` | Add `scripts/ralph-item-reviewer-prompt.md` to orchestrator install list |

## Risks and open questions

- None — the template file already exists, the consumer already expects it at the right path, only the install manifest is missing.

## Progress log

- [ ] Add `scripts/ralph-item-reviewer-prompt.md -> ~/.degacore/scripts/ralph-item-reviewer-prompt.md` to the orchestrator install section of `commands/apply-core.md`
- [ ] Copy `scripts/ralph-item-reviewer-prompt.md` to `~/.degacore/scripts/` for immediate use (deps: 1)

## Decision log

| Decision | Alternatives considered | Rationale |
|----------|------------------------|-----------|
| Add to existing apply-core manifest | Create a separate install script, move file to agents/ | The file is a script-level prompt template consumed by `orch-review.sh` via `${SCRIPT_DIR}/`. It belongs in `scripts/` and the apply-core manifest is the install mechanism. |

## Completion criteria

- [ ] `grep -c 'ralph-item-reviewer-prompt' commands/apply-core.md` returns at least 1
- [ ] `test -f ~/.degacore/scripts/ralph-item-reviewer-prompt.md` exits 0

--- a/commands/apply-core.md
+++ b/commands/apply-core.md
@@ -80,6 +80,7 @@ Files available:
 - `scripts/orch-review.sh`
 - `scripts/orch-engine.sh`
 - `scripts/orch-verify.sh`
+- `scripts/ralph-item-reviewer-prompt.md`
 - `agents/orch-worker.md`
 - `agents/orch-verifier.md`
 - `hooks/orch-done-sync.sh`
@@ -166,6 +167,7 @@ Read and note which of these already exist under `~/.degacore/`:
 - `~/.degacore/scripts/orch-state.sh`
 - `~/.degacore/scripts/orch-review.sh`
 - `~/.degacore/scripts/orch-verify.sh`
+- `~/.degacore/scripts/ralph-item-reviewer-prompt.md`
 - `~/.degacore/scripts/planner-loop.sh`
 - `~/.degacore/scripts/canon-scaffold.sh`
 - `~/.degacore/scripts/canon.sh`
@@ -452,6 +454,7 @@ Write each shell script to `~/.degacore/scripts/`:
 - `scripts/orch-state.sh` -> `~/.degacore/scripts/orch-state.sh`
 - `scripts/orch-review.sh` -> `~/.degacore/scripts/orch-review.sh`
 - `scripts/orch-verify.sh` -> `~/.degacore/scripts/orch-verify.sh`
+- `scripts/ralph-item-reviewer-prompt.md` -> `~/.degacore/scripts/ralph-item-reviewer-prompt.md`
 
 Run `chmod +x ~/.degacore/scripts/orch-*.sh` after writing.
 

